### PR TITLE
Added reserve progress bar

### LINKED ISSUE #3649: Expenditures: As a user I can display a reserve bar on "Unequal" split
[Provided by Adrian via Contractors project management platform]

## Description
When toggle switch is on "Unequal", user has to be able to display reserve bar. A reserve bar will show a percentage usage of max amount of funds declared.

## Acceptance Criteria

- an option to display a bar
- an option to display percent

## Design

- [Figma Link - Reserve percent bar](https://www.figma.com/file/dCtvv76S8mk5HNApjwmBto/Expenditures?node-id=3807%3A285480)

aa36d50cd388052da70b69f2138d6657

### PR DESCRIPTION
**Pull request by agnieszkajarosikloj**
## Description

This PR adds  reserve bar when toggle switch is on "Unequal". A reserve bar will show a percentage usage of max amount.
Screenshot 2022-07-22 at 09 26 16

[Figma Link - Reserve percent bar](https://www.figma.com/file/dCtvv76S8mk5HNApjwmBto/Expenditures?node-id=3807%3A285480)

## TODO
* display the actual percentage on the progress bar after adding recipients

Resolves  #3649

### CODE REVIEW

**Review by willm30 — APPROVED** [review]
Looks good to me 👍

File: src/modules/dashboard/components/ExpenditurePage/Split/SplitUnequal/SplitUnequal.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    id: 'dashboard.ExpenditurePage.Split.SplitUnequal.reserve',
+    defaultMessage: 'Reserve',
+  },
+  amountLabel: {
+    id: 'dashboard.ExpenditurePage.Split.amountLabel',
+    defaultMessage: 'Amount',
+  },
+});
+
+const displayName = 'dashboard.ExpenditurePage.Split';
```
Comment (Julianahlk) [inline src/modules/dashboard/components/ExpenditurePage/Split/SplitUnequal/SplitUnequal.tsx:24]: This has the same displayName as Split.tsx. How about using 'dashboard.ExpenditurePage.SplitUnequal'?

**Review by Julianahlk — APPROVED** [review]
Other than my one comment, everything works as intended 👍

**Review by danbr — APPROVED** [review]
LGTM - well done.

**Review by arrenv — CHANGES_REQUESTED** [review]
@agnieszkajarosikloj looks good, just please update the font size to match the rest, as of the circled.

download (62)

**Pull request merged**